pkg/thing/thing.go

```diff
@@ -291,6 +338,15 @@ func (b *baseBuilder) TimeoutRequestAfter(d time.Duration) Builder {
 	return b
 }
 
+// createKID creates a key ID for a signer
+func createKID(key crypto.Signer) (string, error) {
```

Comment:
I think we should export this function and call it `JWKThumbprint` and mention the spec (https://tools.ietf.org/html/rfc7638) in the description.